requirements.txt

```diff
@@ -0,0 +1,5 @@
+sap-document-classification-client==0.1.8
```

Comment:
Ok I think we should find a way to do this directly from the repo. Otherwise version changes will be required here all the time. Will have a look now.